tests/analyzers/stattests/test_registry.py

```diff
@@ -0,0 +1,61 @@
+import pytest
+
+from evidently.analyzers.stattests import ks_stat_test, chi_stat_test, z_stat_test
+from evidently.analyzers.stattests.registry import get_stattest, stattest, StatTestInvalidFeatureTypeError, \
+    StatTestNotFoundError
+
+
+@stattest("test_stattest", allowed_feature_types=["cat"])
```

Comment:
This code changes global stat tests, maybe better will be move it to a test method, save initial state or remove test functions after the test run?